apstools/devices/lakeshore_controllers.py

```diff
@@ -302,11 +196,11 @@ class LakeShore336Device(Device):
     serial = Component(AsynRecord, "serial", kind="omitted")
 
 
-class LS340_LoopBase(PVPositionerSoftDoneWithStop):  # TODO: check, check, check
+class LS340_LoopBase(PVPositionerSoftDoneWithStop):
     """ Base settings for both sample and control loops. """
 
     # position
-    readback = Component(Signal, value=0)
+    # readback is provided by PVPositionerSoftDoneWithStop
     # TODO: Polar reports: sometimes the setpoint PV is not updated
     setpoint = FormattedComponent(
```

Comment:
setpoint is also provided by PVPositionerSoftDoneWithStop so this is confusing

Reply:
The issue here is that the write/read PVs are different, and that is not "allowed" in PVPositionerSoftDoneWithStop.

Reply:
If you mean the two PVs have different prefixes, it is possible to use full PV names:

```python
pos = PVPositionerSoftDoneWithStop("", readback_pv="A:float1", setpoint_pv="b:parameter", name="pos"))
```

Reply:
I meant for cases where there are different PVs for writing and reading a parameter. For example, for the setpoint,  I would like to use the "OUT1:SP_RBV" as the PV reading the value, and "OUT1:SP" as the write PV. I can do that in `EpicsSignal` with the `write_pv` `kwarg`.

<img width="593" alt="Screen Shot 2022-01-19 at 11 37 23 AM" src="https://user-images.githubusercontent.com/10960331/150184733-19687b36-ea55-4ae2-bca9-eb02afce04b2.png">